utils/python/CIME/utils.py

```diff
@@ -149,8 +149,7 @@ def run_cmd(cmd, ok_to_fail=False, input_str=None, from_dir=None, verbose=None,
     if (arg_stderr is _hack):
         arg_stderr = subprocess.PIPE
 
```

Comment:
May need to discuss this further. If the verbose argument is True but logging.info is off, then logging.info prints nothing when it should. If the opposite is true, then we have the opposite problem.